customTests/azure_cuda_bandwidth.nhc

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
```

Comment:
Are these tested to see if they work and run as expected?

Reply:
This was tested on ND96asr and has the expected 24 GBS BW. i.e
  * || check_cuda_bw 24 passes 
  
  but let me verify the test fails on the right condition

Reply:
Test is working as expected